### DISCUSSION TITLE
Welcome to vue-stripe-checkout Discussions!

### DISCUSSION THREAD

**Discussion by jofftiquez** [jofftiquez/vue-stripe-checkout#152] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SH00KS** [comment]
Hey all! Bit stuck on something so wondering if there is anyone who can help :)

Currently setting up my checkouts like the following. We have 3 packages to choose from and they are all one off payments so not subscriptions as such.

`    <stripe-checkout
      ref="checkoutRef"
      mode="payment"
      :pk="publishableKey"
      :line-items="lineItems"
      :success-url="successURL"
      :cancel-url="cancelURL"
      client-reference-id="cus_XXX"
      customer-email="XXX"
      @loading="v => loading = v"
    />`
    
As I am supplying the stripe customer Id as the 'client-reference-id' I would imagine that it would update the existing customer? However it is creating a new customer in stripe each time. Am I doing something wrong here?
    
   Thank's for your time reading this :)

**Reply by jofftiquez** [reply]
Hello, @SH00KS I understand your problem. Let me look at it first :) Altho, can you delete this one and start a new one [here](https://github.com/vue-stripe/vue-stripe/discussions/new)? Thanks.

**Reply by KLala94** [reply]
is vue-stripe compatible with vue 3?
